test/Interop/Cxx/value-witness-table/custom-virtual-destructors.swift

```diff
@@ -0,0 +1,61 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-cxx-interop -Xcc -fno-rtti)
+//
+// REQUIRES: executable_test
+// Windows doesn't support -fno-rtti.
+// UNSUPPORTED: OS=windows-msvc
+
+import CustomDestructorTypes
+import StdlibUnittest
+
+var CXXDestructorTestSuite = TestSuite("CXXDestructor")
+
+@inline(never)
```

Comment:
Why inline(never)? Inlining should not affect semantics, so I think we should let the optimizer do whatever it wants for the test to be realistic.

Reply:
Fair enough.